docs/source/changelog.rst

```diff
@@ -7,7 +7,15 @@ Changelog
 Array
 +++++
 
-- ``apply_gufunc`` implements keywords `axes`, `axis` and `keepdims` (:pr:`3985`) `Markus Gonser`_
```

Comment:
Just noticed: should we re-include this change?

Reply:
No.  I now handle this during the release.  I actually slightly prefer that people don't introduce changes themselves.  This tends to cause merge conflicts.

Reply:
Thank you for watching out for this though.